k9mail/src/test/java/com/fsck/k9/helper/HtmlSanitizerTest.java

```diff
@@ -89,6 +98,15 @@ public void shouldRemoveMetaRefreshButKeepOtherMetaTags() {
                 "<body>Message</body>" +
                 "</html>";
         assertEquals("<html><head><meta http-equiv=\"content-type\" content=\"text/html; charset=UTF-8\" /></head>" +
-                "<body>Message</body></html>", HtmlSanitizer.sanitize(html));
+                "<body>Message</body></html>", htmlSanitizer.sanitize(html));
+    }
+
+    public static HtmlSanitizer getDummyHtmlSanitizer() {
```

Comment:
This doesn't belong here. Either make it a named class, e.g. `FakeHtmlSanitizer` or `HtmlSanitizerHelper` and include this method.